custodia.spec

```diff
@@ -191,6 +192,24 @@ cp %{buildroot}/%{_bindir}/custodia-cli %{buildroot}/%{_bindir}/custodia-cli-3
 %endif
 
 
+%pre
+getent group custodia >/dev/null || groupadd -r custodia
+if ! getent passwd custodia >/dev/null; then
+     useradd -r -g custodia -d / -s /sbin/nologin -c "User for custodia" custodia
+     if [ -d %{_sysconfdir}/custodia ]; then
+         chown -R custodia:custodia %{_sysconfdir}/custodia
+     fi
```

Comment:
What if a user exist but files are not owned by it ?
mentioning the keytab makes me nervous though, what if the admin had carefully defined permissions (maybe using ACLs or other stuff) ? We would trample over it.

I am thinking whether we should do this automatically at all or rather leave to the admin to set up custodia with a user. Perhaps we could provide an instance creation scripts that offers to create the appropriate user instead of trying to stampeded on existing systems.